res/skins/LateNight/style.qss

```diff
@@ -350,6 +346,7 @@
 }
 
 #KnobLabel {
+  font-family: "Lucida Grande", "Helvetica Neue", Arial, "Open Sans Light";
```

Comment:
``font-family: "Open Sans Light";``

I'm amazed again by QT's font processing if this really works. Because there is no font family ``Open Sans Light``. What we have is the font family ``Open Sans``with different styles available, e.g. ``Regular``, ``Light`` etc. . This is correctly reported to mixxx.log if in developer mode where all ``OpenSans-*.ttf`` report as ``Open Sans`` font-family.
To use one of the available styles of a font, we are supposed to use the ``font-weight`` specifier. Given we have ``OpenSans-Regular.ttf`` and ``OpenSans-Light.ttf`` installed, displaying the light variant works like
```javascript
font-family: "Open Sans";
font-weight: 399;
font-size: font: 10px;
...
```
The font weight number are from 0-900 + normal/bold, details here: https://qt.gitorious.org/qt/qt/source/3f2f403191ea5a38c66b79840e3a33e05827d580:src/gui/text/qfontdatabase.cpp#L141

Apparently, there are bugs with QT 4.8.x, where the font-weight do not work reliable.
[QTBUG-38482 - Font style doesn't follow the font weight (fixed in Qt 5.5)](https://bugreports.qt.io/browse/QTBUG-38482)

http://stackoverflow.com/questions/14901933/is-it-possible-to-specify-more-than-just-regular-and-bold-versions-of-a-font

Nonetheless - Good if this fixes the issues on Linux, as the issue never was with OSX, and we are talking about the fallback font for this skin. Than we only need a confirmation that it works on Windows. 
Id rather add the bold variant of the font instead the light-condensed, which appears unused.